webrtc-stats.html

```diff
@@ -1577,6 +1579,33 @@ <h2>
                   the configuration is considered power efficient.
                 </p>
               </dd>
+              <dt>
+                <dfn>totalAssemblyTime</dfn> of type <span class=
+                "idlMemberType">double</span>
+              </dt>
+              <dd>
+                <p>
+                  Only [= map/exist =]s for video. The sum of the time, in seconds, each video frame takes
+                  from the time the first RTP packet is received (reception timestamp) and to the time
+                  the last RTP packet of a frame is received.
```

Comment:
Does the last RTP packet require a market bit to be set? or what happens if the packet with marker bit is lost? i.e., would it wait for the next RTP timestamp to arrive?

Maybe another way of asking, the timer starts when the first RTP packet with a new timestamp arrives. The timer stops when the RTP packet with the same timestamp and the marker bit set is received OR when an RTP packet with a different timestamp is received.

Reply:
from discussion. Incomplete frames do not count towards this metric